docs/whatsnew/0.1.2.rst

```diff
@@ -0,0 +1,27 @@
+.. _whatsnew_012:
+
+0.1.2 (February 21, 2022)
```

Comment:
Let's change this date to be "TBD" (assuming we aren't planning to release 0.1.2 today?)

Reply:
Changed to TBD in commit https://github.com/pvlib/pvanalytics/commit/3443648756d17fb27f798be6192e01c66b30df7b